clik/external/hal_cpu/source/CMakeLists.txt

```diff
@@ -21,6 +21,12 @@ add_library(hal_cpu SHARED
 
 target_compile_definitions(hal_cpu PUBLIC -DBUILD_HAL_DLL)
 
+if(HAL_CPU_WG_MODE)
+  target_compile_definitions(hal_cpu PUBLIC -DHAL_CPU_WG_MODE=1)
+else()
+  target_compile_definitions(hal_cpu PUBLIC -DHAL_CPU_WI_MODE=1)
+endif()
```

Comment:
Can we change the macro setup to make it more obvious that these are mutually exclusive? Something like `-DHAL_CPU_MODE=HAL_CPU_MODE_WG` vs. `-DHAL_CPU_MODE=HAL_CPU_MODE_WI`?

Reply:
done